src/Configurator/CopyFromPackageConfigurator.php

```diff
@@ -85,20 +91,34 @@ public function copyFile(string $source, string $target)
         if (file_exists($target)) {
             return;
         }
+
+        if (!file_exists($source)) {
+            throw new Exception(sprintf('File "%s" does not exist!', $source));
```

Comment:
I have added this bit here, since it seemed sensible to throw an exception if a source file is missing.

Reply:
I would have used `\LogicException` here as it should never happen.